diff --git a/.bashrc b/.bashrc
index 83fbc95..2135187 100644
--- a/.bashrc
+++ b/.bashrc
@@ -1,5 +1,16 @@
 ## ~/.bashrc: executed by bash(1) for non-login shells.
 
+#
+## adb-ssh-socks5 proxy
+export HTTP_PROXY="socks5h://localhost:1080"
+export http_proxy="socks5h://localhost:1080"
+export HTTPS_PROXY="socks5h://localhost:1080"
+export https_proxy="socks5h://localhost:1080"
+export ALL_PROXY="socks5h://localhost:1080"
+export all_proxy="socks5h://localhost:1080"
+export NO_PROXY="localhost"
+export no_proxy="localhost"
+
 #
 ## internal use
 
diff --git a/.gitconfig b/.gitconfig
index 84d87eb..8fc1729 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -5,6 +5,9 @@
 	email = microsounds@users.noreply.github.com
 	signingkey = ~/.ssh/id_rsa.pub
 
+[http]
+	proxy = socks5h://localhost:1080
+
 [alias]
 	# outline commits made before HEAD
 	summary = past
diff --git a/.local/bin/chromium b/.local/bin/chromium
index bd94cb2..f13764e 100755
--- a/.local/bin/chromium
+++ b/.local/bin/chromium
@@ -91,5 +91,6 @@ fi
 # 08/2024: disable UA spoofing because it triggers cloudflare filters on many sites
 /usr/bin/chromium \
 	--user-data-dir="$CONF" --disk-cache-dir="$CACHE" \
+	--proxy-server='socks5://localhost:1080' \
 	"$@"
 #	--user-agent="$UA_STRING" "$@"
diff --git a/.ssh/config b/.ssh/config
new file mode 100644
index 0000000..0985493
--- /dev/null
+++ b/.ssh/config
@@ -0,0 +1,2 @@
+Host *
+	ProxyCommand nc -x localhost:1080 -X 5 %h %p
