# change monocle window counter appearance
# for dwm 6.2

diff --git a/dwm.c b/dwm.c
index 4465af1..35a40c1 100644
--- a/dwm.c
+++ b/dwm.c
@@ -1110,7 +1110,7 @@ monocle(Monitor *m)
 		if (ISVISIBLE(c))
 			n++;
 	if (n > 0) /* override layout symbol */
-		snprintf(m->ltsymbol, sizeof m->ltsymbol, "[%d]", n);
+		snprintf(m->ltsymbol, sizeof m->ltsymbol, "%d窓", n);
 	for (c = nexttiled(m->clients); c; c = nexttiled(c->next))
 		resize(c, m->wx, m->wy, m->ww - 2 * c->bw, m->wh - 2 * c->bw, 0);
 }
